guide/interactions/modals.md

```diff
@@ -92,29 +92,50 @@ client.on('interactionCreate', async interaction => {
 });
 ```
 
-
 Restart your bot and invoke the `/ping` command again. You should see a popup form resembling the image below:
 
 <img width=450 src="./images/modal-example.png">
 
 ## Receiving modal submissions
 
-Modals are received via an interaction. You can check if a given interaction is a modal by invoking the `#isModalSubmit()` method:
+Modals are received via an interaction. You can check if a given interaction is a modal by checking if the interaction is a `ModalSubmit`.
+
+```js {1,4}
+const { InteractionType } = require('discord.js');
 
-```js {2}
 client.on('interactionCreate', interaction => {
-	if (!interaction.isModalSubmit()) return;
+	if (interaction.type !== InteractionType.ModalSubmit) return;
 	console.log(interaction);
 });
 ```
 
+## Responding to modal submissions
+
+The `ModalSubmitInteraction` class provides the same methods as the `CommandInteraction` class. These methods behave equally:
+
+- `reply()`
+- `editReply()`
+- `deferReply()`
+- `fetchReply()`
+- `deleteReply()`
+- `followUp()`
+
+```js{1,4-6}
+client.on('interactionCreate', interaction => {
+	if (interaction.type !== InteractionType.ModalSubmit) return;
+	if (interaction.customId === 'myModal') {
+		await interaction.reply({ content: 'Your submission was recieved successfully!' });
+	}
+});
+```
+
 ## Extracting data from modal submissions
 
 You'll most likely need to read the data sent by the user in the modal. You can do this by accessing the `#fields` instance field on the interaction. From there you can call `#getTextInputValue` with the custom id of the text input to get the value.
 
-```js{4-8}
+```js
```

Comment:
The above text talked about accessing the fields and text input values. The relevant lines should be highlighted here.